setup.py

```diff
@@ -67,7 +68,7 @@
         "color-operations",
         "flask",
         "flask_cors",
-        "marshmallow>=3.0.0",
+        "marshmallow>=3.0.0,<4",
```

Comment:
I don't think that's needed. main is already compatible with 4.0. Let me rebase this.